modules/gistic2/1.0/gistic2.smk

```diff
@@ -0,0 +1,304 @@
+                + input_dir + "capture--projection/all--" + wildcards.projection + ".seg"
+    elif "capture" in seq_types and "genome" not in seq_types:
+        param = "--capture " + input_dir + "capture--projection/all--" + wildcards.projection + ".seg"
+    elif "capture" not in seq_types and "genome" in seq_types:
+        param = "--genome " + input_dir + "genome--projection/all--" + wildcards.projection + ".seg"
+    
+    return(param)
+
+# Merges capture and genome seg files if available, and subset to the case_set provided
+rule _gistic2_prepare_seg:
+    input:
+        seg = expand(str(rules._gistic2_input_seg.output.seg),
+                    allow_missing=True,
+                    seq_type=CFG["samples"]["seq_type"].unique()
+                    ),
+        all_sample_sets = ancient(str(rules._gistic2_input_sample_sets.output.all_sample_sets)),
+        seg_dir = CFG["dirs"]["inputs"]
+    output:
+        seg = CFG["dirs"]["prepare_seg"] + "{case_set}--{projection}.seg"
+    log:
+        stdout = CFG["logs"]["prepare_seg"] + "{case_set}--{projection}.stdout.log",
+        stderr = CFG["logs"]["prepare_seg"] + "{case_set}--{projection}.stderr.log"
+    group: 
+        "input_and_format"
+    params:
+        script = CFG["prepare_seg"],
+        case_set = CFG["case_set"],
+        seg_input_params = _get_seg_input_params
+    shell:
+        op.as_one_line("""
+        Rscript {params.script} 
+        {params.seg_input_params} 
+        --projection {wildcards.projection} 
+        --output_dir $(dirname {output.seg})/ 
+        --all_sample_sets {input.all_sample_sets} 
+        --case_set {params.case_set} 
+        > {log.stdout} 2> {log.stderr}
+        """)
+
+# Create a markers file that has every segment start and end that appears in the seg file
+rule _gistic2_make_markers:
+    input:
+        seg = str(rules._gistic2_prepare_seg.output.seg)
+    output:
+        temp_markers = temp(CFG["dirs"]["markers"] + "temp_markers--{case_set}--{projection}.txt"),
+        markers = CFG["dirs"]["markers"] + "markers--{case_set}--{projection}.txt"
+    log:
+        stderr = CFG["logs"]["markers"] + "{case_set}--{projection}--markers.stderr.log"
+    shell: 
+        op.as_one_line("""
+        sed '1d' {input.seg} | cut -f2,3 > {output.temp_markers}
+            &&
+        sed '1d' {input.seg} | cut -f2,4 >> {output.temp_markers}
+            &&
+        sort -V -k1,1 -k2,2nr {output.temp_markers} | uniq | nl > {output.markers} 
+        2> {log.stderr}
+        """)
+
+# Run gistic2 for a single seq_type (capture, genome) for the confidence thresholds listed in the config
+rule _gistic2_run:
+    input:
+        seg = str(rules._gistic2_prepare_seg.output.seg),
+        refgene_mat = str(rules._gistic2_download_ref.output.refgene_mat),
+        markers = str(rules._gistic2_make_markers.output.markers)
+    output:
```

Comment:
I agree. I like the idea of listing only a few outputs and symlinking them. Are there other meaningful outputs besides:

- scores.gistic
- all_data_by_genes.txt
- all_lesions.conf_90.txt

Reply:
Thanks!!
Yes I think it will be good to also add

- amp_genes.conf_90.txt
- del_genes.conf_90.txt

These are what is used for maftools processing if someone decides to go ahead with that way of downstream analysis. Although there are problems with it, I can imagine there will be users who would prefer to do that and at least it will be convenient to have all in one place.